x/pairing/genesis.go

```diff
@@ -37,9 +43,48 @@ func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 	genesis := types.DefaultGenesis()
 	genesis.Params = k.GetParams(ctx)
 
-	genesis.UniquePaymentStorageClientProviderList = k.GetAllUniquePaymentStorageClientProvider(ctx)
-	genesis.ProviderPaymentStorageList = k.GetAllProviderPaymentStorage(ctx)
-	genesis.EpochPaymentsList = k.GetAllEpochPayments(ctx)
+	epochs, uniqueEpochSessions := k.GetAllUniqueEpochSessionStore(ctx)
```

Comment:
lets make it a list of one object and not rely on the lists orderr

Reply:
done. see https://github.com/lavanet/lava/commit/ebb7dc05fbcd023b4d71351daf090150b9322ca1